composer.json

```diff
@@ -44,6 +44,8 @@
         "mockery/mockery": "1.0.*@dev",
         "fzaninotto/faker": "1.2.*",
         "codeclimate/php-test-reporter": "dev-master",
-        "johnkary/phpunit-speedtrap": "~1.0@dev"
+        "johnkary/phpunit-speedtrap": "~1.0@dev",
+        "fabpot/php-cs-fixer": "^1.10",
+        "phpmd/phpmd": "^2.2"
```

Comment:
@yitznewton 

Just as an FYI, I have contributed a small feature to `composer/composer`, the `--sort-packages` flag!

Not sure if you are as pedantic as I am, but I like it, it keeps platform requirements and dependencies sorted alphabetically:

```
$ composer require --dev --sort-packages fabpot/php-cs-fixer:^1.10
```